JavaFiddle-ejb/src/main/java/ru/javafiddle/core/ejb/UserBean.java

```diff
@@ -22,10 +25,13 @@
 
     private static final String DEFAULT_USER_STATUS = "registered";
 
-    @PersistenceContext
-    EntityManager em;
+    @Resource
+    private SessionContext ctx;
 
-    UserBean(){}
+    @PersistenceContext(name = "JFPersistenceUnit")
+    private EntityManager em;
+
+    public UserBean(){}
```

Comment:
Checkstyle remark:
Recommended code style for brackets is 
public SomeClass() {

}